tests/admin/test-class-plugin-suggestions.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+require_once WPSEO_TESTS_PATH . 'admin/test-class-wpseo-suggested-plugins-double.php';
```

Comment:
Maybe it's better to include this in the `setUp` method. In this situation including this file might have side effects.